code/modules/unit_tests/map_tests.dm

```diff
@@ -112,9 +112,10 @@
 
 /datum/map_per_tile_test/nearspace_checker
 	var/allowed_turfs = list(/turf/space,
-							 /turf/simulated/floor/plating/airless,
-							 /turf/simulated/wall
-							)
+		/turf/simulated/floor/plating/airless,
+		/turf/simulated/floor/plasteel/airless,
+		/turf/simulated/wall
+		)
```

Comment:
This one level in.

Reply:
Like this? I suck at spacing out lists properly.
```
var/allowed_turfs = list(/turf/space,
		/turf/simulated/floor/plating/airless,
		/turf/simulated/floor/plasteel/airless,
		/turf/simulated/wall
	)
```

Reply:
```
var/allowed_turfs = list(/turf/space,
	/turf/simulated/floor/plating/airless,
	/turf/simulated/floor/plasteel/airless,
	/turf/simulated/wall
)
```

Reply:
Yeah perfect